app/src/main/java/io/appium/uiautomator2/handler/AppStrings.java

```diff
@@ -5,20 +5,20 @@
 import java.io.FileInputStream;
 import java.io.IOException;
 
-import io.appium.uiautomator2.handler.request.BaseRequestHandler;
+import io.appium.uiautomator2.handler.request.SafeRequestHandler;
 import io.appium.uiautomator2.http.AppiumResponse;
 import io.appium.uiautomator2.http.IHttpRequest;
 import io.appium.uiautomator2.server.WDStatus;
 import io.appium.uiautomator2.utils.Logger;
 
-public class AppStrings extends BaseRequestHandler {
+public class AppStrings extends SafeRequestHandler {
     public AppStrings(String mappedUri) {
         super(mappedUri);
     }
 
     @Override
-    public AppiumResponse handle(IHttpRequest request) {
-        String jsonString = "";
+    public AppiumResponse safeHandle(IHttpRequest request) {
+        String jsonString;
         final String filePath = "/data/local/tmp/strings.json";
 
```

Comment:
Java 7 feature ,try with resources is not allowing to compile the code. Since we have
 we have **minSdk** level 18 because our driver will support for **android API level 18** and above. If our minSdk is 19 then we would have used it.
https://github.com/appium/appium-uiautomator2-server/blob/master/app/build.gradle#L26

http://stackoverflow.com/questions/24290830/try-with-resources-are-not-supported-at-this-language-level-android